packages/kit/src/core/build/index.js

```diff
@@ -349,7 +349,8 @@ async function build_server(
 					ssr: ${s(config.kit.ssr)},
 					target: ${s(config.kit.target)},
 					template,
-					trailing_slash: ${s(config.kit.trailingSlash)}
+					trailing_slash: ${s(config.kit.trailingSlash)},
+					cspNonce: ${s(config.kit.cspNonce)}
```

Comment:
We use snake_case for internal variables
```suggestion
					csp_nonce: ${s(config.kit.cspNonce)}
```